### DISCUSSION TITLE
Monthly ESMValTool community meeting - December 2023

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#138] [Meetings]
The next online community meeting will be on

December 11th at 2pm UTC / 3pm CET

The duration will be one hour.  [Join the meeting](https://ncas.zoom.us/j/94499149982?pwd=STVyVThFMkZmMWJQNHNPYlJjMDRKdz09)

Calendar invitations to the meeting are sent to the mailing list, join it [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

**Format and agenda**

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #132

**Comment by paultgriffiths** [comment]
Updates
@rswamina - will work on tutorial and user engagement
@katjaweigel - will work on PRs, including IPCC plot PR.  When is next release?  Would help with assessing PRs.
@valeriupredoi - should be real soon now, maybe this week.
@ehogan - two new contributors joining from UKMO - Emma, please add Github handles for Ed (@mo-gill) and Chris (@chrisbillowsMO)? Done! 😊 
@ehogan  - New CMEW tool to replace AutoAssess - CMEW is a combination of CDDS and ESMValTool for model evaluation and assessment.  Going public in 2024.  Possible workshop?
@headmetal - working on integrating ESMValTool into ACCESS-NRI workflow.  see [here](https://github.com/ACCESS-NRI/ESMValTool-workflow) and an annoucement [here](https://forum.access-hive.org.au/t/esmvaltool-workflow-v1-0-launches-today-here-is-what-you-need-to-know/1599)
@rswamina -  alternating meetings at more US- and Australia-friendly times.  @ESMValGroup/esmvaltool-developmentteam - is there anyone who'd like to host?

**Reply by valeriupredoi** [reply]
@paultgriffiths cheers :beer: The Met Office thingie is CMEW (like the :cat: says :grin:)

**Reply by valeriupredoi** [reply]
@ehogan pronounces it as see-myoo but I keep insisting it should be pronounced s- :cat: On that note, happy holidays and a good new year, folks!

**Reply by catherinehardacre** [reply]
@rswamina - Yes please to the odd meeting at Australia friendly times!

**Reply by rbeucher** [reply]
+1 for meetings at a more Oz friendly time. The middle of the night is way too hard for me 😁

**Reply by ehogan** [reply]
CMEW = Climate Model Evaluation Workflow. Pronounced like see-mew (which should sound like a cat, sorry if it doesn't when I say it!) 😝

**Reply by valeriupredoi** [reply]
@catherinehardacre surely you meant Kiwi time :kiwi:

**Reply by catherinehardacre** [reply]
@valeriupredoi Obviously kiwi time would be superior, but my expectations are realistic ;)

**Reply by paultgriffiths** [reply]
@rbeucher  and @catherinehardacre  - do either of you fancy hosting the odd meeting?

**Reply by rbeucher** [reply]
@paultgriffiths sure, we could do that! Good idea.

**Reply by catherinehardacre** [reply]
@paultgriffiths Sure! Having made my demands, I should step up :) See you in 2024...

**Reply by paultgriffiths** [reply]
@catherinehardacre  and @rbeucher  - how would you feel about organising the Feb 2024 meeting to be at a time convenient for you guys?

**Reply by rbeucher** [reply]
Yes we can do that. @flicj191 should be able to help. Is there a process that has been documented somewhere? 
Happy to coordinate with @catherinehardacre.

**Reply by paultgriffiths** [reply]
Happy to help you to arrange this if you would like.  I should say that January’s meeting suggested that we could aim for the February call to be on SH-friendly time - if you start a discussion for the February meeting in the Discussions section and create a doodle, I can send it to the ESMValTool mailing list (if that helps).  If February doesn’t work for you guys, it can be March!

**Reply by rbeucher** [reply]
@paultgriffiths I have started a discussion here #144. I have created a doodle poll with more friendly times for us in the Pacific region.
Let me know what you think and if we can post the invite to the mailing list.
